doc/api/util.md

```diff
@@ -398,6 +398,10 @@ stream.write('With ES6');
 <!-- YAML
 added: v0.3.0
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/30768
+    description: Prototype properties will be inspected from now in case
+                 `showHidden` is truthy.
```

Comment:
```suggestion
                 `showHidden` is `true`.
```

Even if this practically checks for truthiness, the documentation says `boolean` and we shouldn't encourage other types.